src/components/Button/Button.tsx

```diff
@@ -92,17 +94,26 @@ class Button extends UIComponent<any, any> {
     accessibility: ButtonBehavior as Accessibility,
   }
 
-  public renderComponent({ ElementType, classes, accessibility, rest }): React.ReactNode {
+  public state = {
+    isLastFocusFromMouse: false,
+  }
+
+  public renderComponent({ ElementType, classes, accessibility, styles, rest }): React.ReactNode {
     const { children, content, disabled, iconPosition } = this.props
+    const { isLastFocusFromMouse } = this.state
+    console.log('state is', isLastFocusFromMouse)
+
     const hasChildren = childrenExist(children)
 
     return (
       <ElementType
         className={classes.root}
+        styles={isLastFocusFromMouse ? { root: styles.focusFromMouse } : undefined}
```

Comment:
Does `styles` attribute work for html elements? I thought it works for Components only.
For html element it results in `<button ... styles="[object Object]" ...>`